man/man5/zfs-module-parameters.5

```diff
@@ -250,6 +250,25 @@ Use \fB1\fR for yes (default) and \fB0\fR for no.
 .sp
 .ne 2
 .na
+\fBparallel_vdev_open\fR (bool)
```

Comment:
Thanks for updating the name, but it looks like you overlooked updating the man page.